validator/testcases/javascript/instanceactions.py

```diff
@@ -422,6 +423,18 @@ def os_file_readTo_callback(argument, traverser, node, wrapper):
         tier=5)
 
 
+def quote_callback(argument, traverser, node, wrapper):
+    traverser.warning(
+        err_id=("js", "string", "quote"),
+        warning="The quote interface has been removed",
+        description="The quote interface has been removed. See %s for more "
```

Comment:
This should probably mention that the interface was removed from `String.prototype`.